website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Metadata API Overview"
+id: "metadata-overview"
+---
+
+Every time that dbt Cloud runs a dbt project, it generates metadata which pertains to the accuracy, recency, configuration, and structure of the views and tables in the warehouse. dbt Cloud serves a GraphQL API which supports arbitrary queries over this metadata; the endpoint for this API is `https://metadata.cloud.getdbt.com/graphql`. This API is an incredibly rich resource for evaluating data health longitudinally or at a point in time. Please note that today, any query on our metadata API today will refer to the last run.
```

Comment:
we're missing an i in this url, should be:
https://metadata.cloud.getdbt.com/graphiql